src/test/java/teammates/test/cases/ui/InstructorFeedbackQuestionSubmissionEditPageActionTest.java

```diff
@@ -31,6 +31,7 @@ public static void classSetUp() throws Exception {
 
     @Test
     public void testExecuteAndPostProcess() throws Exception {
+        @SuppressWarnings("unused")
         AccountAttributes instructorAccount = dataBundle.accounts.get("instructor1OfCourse1");
```

Comment:
You're right, it was changed but it is never used, I was mistaken and thought that it was changed and used, removing it in the coming commit.